content/21.nutraceuticals.md

```diff
@@ -161,7 +161,8 @@ Post-infection, these levels return to baseline ranges [@doi:10.1177/00369330730
 It has been shown that as little as 0.1 g/d of vitamin C can maintain normal plasma levels of vitamin C in healthy individuals, but higher doses of at least 1-3 g/d are required for critically ill patients in ICUs [@doi:10.3390/nu11040708].
 Indeed, vitamin C deficiency appears to be common among COVID-19 patients [@doi:10.1016/j.medidd.2020.100064; @doi:10.1186/s13054-020-03249-y].
 COVID-19 is also associated with the formation of microthrombi and coagulopathy [@doi:10.1172/JCI140183] that contribute to its characteristic lung pathology [@doi:10.1183/16000617.0240-2020], but these symptoms can be ameliorated by early infusions of vitamin C to inhibit endothelial surface P-selectin expression and platelet-endothelial adhesion [@doi:10.3390/antiox6030049].
-Intravenous vitamin C also reduced D-dimer levels, which are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74], in a case study of 17 COVID-19 patients [@doi:10.1080/14787210.2020.1794819].
+Intravenous vitamin C also reduced D-dimer levels in a case study of 17 COVID-19 patients [@doi:10.1080/14787210.2020.1794819].
+D-dimer levels are an important indicator of thrombus formation and breakdown, which are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74]. 
```

Comment:
```suggestion
D-dimer levels are an important indicator of thrombus formation and breakdown and are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74]. 
```
I think otherwise it sounds like this is saying that thrombus formation & breakdown are noticably elevated